runtime/gc_modron_startup/mmparseXgcpolicy.cpp

```diff
@@ -85,7 +85,7 @@ isMetronomeGCPolicySupported(MM_GCExtensions *extensions)
 #if defined(J9VM_GC_REALTIME)
 #if defined(AIXPPC)
 	return true;
-#elif defined(LINUX) && (defined(J9HAMMER) || defined(J9X86))
+#elif defined(LINUX) && (defined(J9HAMMER) || defined(J9X86) || defined(J9RISCV64))
```

Comment:
Metronome GC policy is very restricted for platforms (AIX and Linux x86)
I would not add RISC-V to this list (at least for now)

Reply:
Already removed as suggested above.